lib/actions/ssh.coffee

```diff
@@ -92,7 +92,7 @@ module.exports =
 			Promise.props
 				username: resin.auth.whoami()
 				uuid: device.uuid		# get full uuid
-				containerId: resin.models.device.getApplicationInfo(device.uuid).get('containerId')
+				containerId: resin.models.device.getApplicationInfo(device.uuid).then(JSON.parse).get('containerId')
```

Comment:
Nice find @pimterry! @Page- and @emirotin are the go-to experts for anything API-related. The fix looks simple; iirc there's a `proxy()` middleware function in the API code that handles request proxying to the device supervisor and my guess is this is the culprit wrt incorrect content-type reponses. But let's take this back to the team for more discussion, because I'm not sure if changing the response type will break anything else (e.g. the `python-sdk` ?)

If you feel like diving into the API feel free to send a PR, but in any case, let's create an issue there to make sure this doesn't slip through.